autosubmit/autosubmit.py

```diff
@@ -2396,6 +2404,12 @@ def run_experiment(expid, notransitive=False, start_time=None, start_after=None,
                         exp_history.finish_current_experiment_run()
                     except Exception:
                         Log.warning("Database is locked")
+                # Create rocrate object if requested
+                rocrate_data = as_conf.experiment_data.get("ROCRATE", None)
+                if rocrate_data:
+                    Autosubmit.provenance(expid, rocrate=True)
+                else:
+                    print("rocrate.yml not found in CONFIG. Can't create rocrate object. ")
```

Comment:
I think this message needs to be changed? `rocrate.yml` is not necessarily the name of the file that will contain `ROCRATE`, and `CONFIG` too exists in our YAML but I think `ROCRATE` is not under it.

And we should use a log instead of `print`, as the `print` will always be written, so developers/users cannot control it. Whereas a logger normally offers users features to suppress certain levels, disable it completely, make it more verbose, etc.

Maybe this:

```suggestion
                rocrate_data = as_conf.experiment_data.get("ROCRATE", None)
                if rocrate_data:
                    Autosubmit.provenance(expid, rocrate=True)
                else:
                    Log.info("ROCRATE not present in experiment YAML configuration. No RO-Crate archive created.")
```